devtools/common/src/main/java/io/quarkus/BasicRest.java

```diff
@@ -167,8 +167,8 @@ private void createDockerFile() throws IOException {
     }
 
     private void createMicroProfileConfig() throws IOException {
-        File meta = new File(projectRoot, "src/main/resources/META-INF");
-        File file = new File(mkdirs(meta), "microprofile-config.properties");
+        File meta = new File(projectRoot, "src/main/resources");
+        File file = new File(mkdirs(meta), "application.properties");
         if (!file.exists()) {
             Files.write(file.toPath(), Arrays.asList("# Configuration file", "key = value"), StandardOpenOption.CREATE_NEW);
```

Comment:
Unrelated but I think maybe we should add a `# ` before `key = value`. If at some point, we want to check the configuration is properly consumed, we will have this property defined in all the generated projects.

Reply:
That's a great idea! I'll add it

Reply:
Done